mypy/main.py

```diff
@@ -31,6 +33,40 @@ def __init__(self) -> None:
         self.dirty_stubs = False
 
 
+class MyPyCommand(setuptools.Command):
+    """The :class:`MyPyCommand` class is used by setuptools to perform
+    type checks on registered modules.
+    """
+
+    description = "Validate PEP 0484 Types with mypy"
+    user_options = [
+        ('mypy-args=', None, 'Arguments to pass through to mypy')
+    ]  # type: List[tuple]
+
+    def initialize_options(self):
+        self.mypy_args = ''
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        try:
+            mypy_options = []
+            for package in self.distribution.packages:
```

Comment:
I believe this won't currently work if there's more than one -- @JukkaL, do I have that right, or am I out of date on that?

If that is the case, let's check that we have only one package in the distribution and give an error if more than one.